helm-charts/stable/cloudwatch-agent/Chart.yaml

```diff
@@ -0,0 +1,36 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
We are technically in the new year, might be worth changing to 2021? I don't know how these years work though